### ISSUE TITLE
issue on android

### ISSUE THREAD

**Issue by rajeshscontact** [jasonslyvia/react-lazyload#132]
When lazy loading more than 500 items in android, the screen freezes periodically

**Comment by jasonslyvia** [comment]
Hi, thanks for reporting this issue, can you confirm this behavior is caused by react-lazyload? For example, try removing `<LazyLoad>` and see what happens.

**Comment by jasonslyvia** [comment]
Additional environment info is also welcomed, like what os version, what browser, what lazyload version.

**Comment by rajeshscontact** [comment]
Yes, I tried without LazyLoad, the app was loading on android after a while (delay for Dom to render all components).

Version: "react-lazyload": "^2.3.0"
OS: Android
Browser: chrome v57.0.2987

**Comment by jackYouth** [comment]
I have a similar problem

**Comment by rubencodes** [comment]
I was seeing similar behavior, adding the `debounce` prop helped.
